src/hpx_wrap.cpp

```diff
@@ -23,10 +23,13 @@ namespace hpx_start
     HPX_SYMBOL_EXPORT bool include_libhpx_wrap __attribute__((weak)) = false;
 }
 
+#include <hpx/hpx_finalize.hpp>
 #include <hpx/hpx_init.hpp>
-#include <hpx/hpx.hpp>
+#include <hpx/runtime/runtime_mode.hpp>
+#include <hpx/util/function.hpp>
 
 #include <iostream>
```

Comment:
Do we need `<iostream>` for this file? Looks like a left-over.

Reply:
Good catch! Probably not.

Reply:
Removed.